### PR TITLE
MHP-1611: Add user_created: false filter to all organization endpoints.

### PR DESCRIPTION
**Pull request by OzzieOrca**
Hopefully there aren't any more places we need to do this. It's really hard to find API calls in this codebase...

### CODE REVIEW

**Comment by OzzieOrca** [comment]
```bash
curl 'https://api-stage.missionhub.com/apis/v4/people/me?include=user,organizational_permissions.organization' -H 'Accept: application/json, text/plain, */*' -H 'Referer: https://localhost:8080/ministries/root' -H 'Origin: https://localhost:8080' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Authorization: Bearer xxxxxx' --compressed
```
returns 17 orgs.

```bash
curl 'https://api-stage.missionhub.com/apis/v4/organizations?filters%5Buser_created%5D=false&include=organizational_permissions' -H 'Accept: application/json, text/plain, */*' -H 'Referer: https://localhost:8080/ministries/root' -H 'Origin: https://localhost:8080' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Authorization: Bearer xxxxxx' --compressed
```
returns 25 orgs

Not sure why there's a difference there. Maybe we need to look into the Rails filters for those endpoints before pushing this further.

Basically I needed to rewrite https://github.com/CruGlobal/missionhub-web/pull/11/files#diff-70c944e86db1d88ae5e7075522257380L18. `JsonApiDataStore.store.findAll('organizational_permission')` takes what what was loaded by that top curl and runs with it. Does the org endpoint filter orgs by the user having an org permission with it and that org permission being user or admin https://github.com/CruGlobal/missionhub-web/blob/sunset-legacy/app/assets/javascripts/angular/services/permission.service.js#L9?

Does this look correct?

**Comment by OzzieOrca** [comment]
Adding `filters[descendants]` fixed the data issue I was having above.

File: app/assets/javascripts/angular/services/loggedInPerson.service.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function loggedInPerson(
             .get(
                 modelsService.getModelMetadata('person').url.single('me'),
                 {
-                    include: 'user,organizational_permissions.organization',
+                    include: 'user,organizational_permissions',
```
Comment (dsgoers) [inline app/assets/javascripts/angular/services/loggedInPerson.service.js]: Why did you add `organization?`

Comment (OzzieOrca) [reply app/assets/javascripts/angular/services/loggedInPerson.service.js]: You mean why did I remove it?

File: app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ function myPeopleDashboardService(httpProxy, modelsService, _) {
                         {
                             sort: '-active_people_count',
                             include: '',
+                            'page[limit]': 250,
```
Comment (dsgoers) [inline app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js]: The limit on the mobile app is 100.  Idk which is better, just letting you know.

File: app/assets/javascripts/angular/components/myOrganizationsDashboard/myOrganizationsDashboard.service.js [hunk 3]
```diff
@@ -9,26 +9,26 @@ function myOrganizationsDashboardService(
...
-                        permission.person_id === loggedInPerson.person.id &&
-                        _.includes(
-                            permissionService.adminAndUserIds,
-                            permission.permission_id,
-                        )
-                    );
-                })
-                .map('organization')
-                .value();
+            return httpProxy
```
Comment (dsgoers) [inline app/assets/javascripts/angular/components/myOrganizationsDashboard/myOrganizationsDashboard.service.js]: Were the orgs loaded elsewhere prior to this change?

Comment (OzzieOrca) [reply app/assets/javascripts/angular/components/myOrganizationsDashboard/myOrganizationsDashboard.service.js]: They were loaded here before https://github.com/CruGlobal/missionhub-web/pull/11/files#diff-bd8e60d6826d256a0c531de2cb066f5fL19. As part of the me user request on app load. Which  we couldn't filter.

File: app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js (hunk 2 above)
Comment (OzzieOrca) [reply app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js]: I just picked a number. I can lower it since that might help prevent performance issues and make it consistent. I'll make a ticket for pagination since I haven't done that yet.

File: app/assets/javascripts/angular/services/loggedInPerson.service.js (hunk 1 above)
Comment (dsgoers) [reply app/assets/javascripts/angular/services/loggedInPerson.service.js]: Yes lol

**Comment by dsgoers** [comment]
gotcha

File: app/assets/javascripts/angular/components/myOrganizationsDashboard/myOrganizationsDashboard.service.js [hunk 4]
```diff
@@ -9,26 +9,26 @@ function myOrganizationsDashboardService(
...
+                .get(
+                    modelsService.getModelMetadata('organization').url.all,
+                    {
+                        'filters[user_created]': false,
+                        'filters[descendants]': false,
+                        include: 'organizational_permissions',
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/myOrganizationsDashboard/myOrganizationsDashboard.service.js]: This needs a limit right? I missed it earlier...

File: app/assets/javascripts/angular/services/loggedInPerson.service.js (hunk 1 above)
Comment (OzzieOrca) [reply app/assets/javascripts/angular/services/loggedInPerson.service.js]: If you wanted an answer, the answer is we're loading it elsewhere now (on one of the other lines you commented on).

**Comment by OzzieOrca** [comment]
Thanks :)
